recipes/gffcompare/build.sh

```diff
@@ -1,9 +1,17 @@
 #!/bin/bash
 
+set -xe
+
 export CXX="$CXX"
 export LINKER="$CXX"
 
+# GCLIB needs gcc to detect its version
+# GCLIB itself is git-cloned by gffcompare's Makefile
+ln -s ${CC} $BUILD_PREFIX/bin/gcc
```

Comment:
Just out of curiosity, why do you need to do this? Isn't gcc already in PATH?

Reply:
I should remove this now (after removing the build for GCLIB).
gclib's makefile [uses](https://github.com/gpertea/gclib/blob/03aab388c2c2a575ae37f51216efdc8fb4c5b30f/Makefile#L55) `gcc` while Conda provides `aarch64-conda-linux-gnu-cc`